content/300-accelerate/600-faq.mdx

```diff
@@ -8,6 +8,18 @@ toc: true
 
 <TopBlock>Below are frequently asked questions about Accelerate.</TopBlock>
 
+## When do I have to enable static IP for Prisma Accelerate?
+
+A static IP address is an IPv4 or an IPv6 address that doesn’t change. Unlike dynamic IP addresses, which can change unpredictably, traffic from static IP addresses are predictable. The predictability of traffic originating from static IP addresses allows you to precisely identify traffic sources. IP addresses need to be predictable when implementing security measures such as IP allowlisting or firewall configurations to restrict database access to trusted sources only.
```

Comment:
> traffic from static IP addresses are predictable

Traffic is predictable? That doesn't sound right. Not sure what we're trying to say here.

```suggestion
A static IP address is an IPv4 or an IPv6 address that is fixed. Unlike dynamic IP addresses, which can change unpredictably, traffic from static IP addresses can be easily identified. This allows the implementation of various security measures, such as IP allowlisting, so database access can be limited to trusted sources only.
```

Reply:
Can we simplify this section with;

Enable Static IP for Accelerate when your security setup requires IP allowlisting or if you're implementing firewalls that only permit access from trusted IPs, ensuring controlled and secure database connections.

Reply:
@luanvdw I also wanted to tell our users what the static IPs are and it's usefulness. While I like your suggestion, could we have something like this?

```suggestion
A static IP address is an IPv4 or an IPv6 address that is fixed. Unlike dynamic IP addresses, which can change unpredictably, traffic from static IP addresses can be easily identified. Enable static IP for Accelerate when your security setup requires IP allowlisting or if you're implementing firewalls that only permit access from trusted IPs, ensuring controlled and secure database connections.
```